app/controllers/volunteers_controller.rb

```diff
@@ -66,6 +66,37 @@ def remove
     # for volunteer to remove project, perhaps include a note for why they're
     # no longer available?
   end
+  
+  def add_repairs
+    @repairs = Repair.all
+    # @repairs = Repair.find(params[:id])
+    @volunteer = current_user.volunteer
```

Comment:
There is a potential problem here in that @volunteer may be nil if the volunteer's profile is not set up.  In which case you would redirect to the new volunteer page.